packages/roosterjs-editor-plugins/test/TableResize/tableResizeTest.ts

```diff
@@ -67,13 +67,13 @@ describe('TableResize plugin tests', () => {
 
     it('removes the vertical inserter when moving the cursor out of the offset zone', () => {
         const rect = initialize();
-        runTest({ x: rect.right, y: rect.top }, { x: rect.right / 2, y: rect.bottom }, false);
+        runTest({ x: rect.right, y: rect.top - 2 }, { x: rect.right / 2, y: rect.bottom }, false);
```

Comment:
make it a constant instead put "2" everywhere.

Reply:
Done